src/octopeer-github/content/ElementSelectionBehaviour/GenericElementSelectionBehaviour.ts

```diff
@@ -49,9 +49,9 @@ class GenericElementSelectionBehaviour implements ElementSelectionBehaviour {
      */
     public getCallback(eventID: EventID) {
         if (this.data.callback === undefined) {
+            const self = this;
             return (eventObject: JQueryEventObject) => {
-                this.database.postSemantic(EventFactory.semantic(this.getElementID(), eventID, new Date().getTime(), 0),
-                    EMPTY_CALLBACK, EMPTY_CALLBACK);
+                self.database.postSemantic(EventFactory.semantic(self.getElementID(), eventID), EMPTY_CALLBACK, EMPTY_CALLBACK);
```

Comment:
Can you please change `self` back to `this`? in the lambda notation, `this` is not overridden.